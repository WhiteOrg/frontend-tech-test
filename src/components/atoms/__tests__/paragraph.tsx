import React from "react";
import renderer from "react-test-renderer";
import Paragraph from "../paragraph";

describe("ATOMS: Render testing", () => {
  test("Should render paragraph", () => {
    const component = renderer.create(
      <div>
        <Paragraph size="small" color="#17dfe7">
          PRIZE POOL
        </Paragraph>
        <Paragraph size="medium" bold>
          P.C
        </Paragraph>
        <Paragraph size="large" color="white">
          £25,000.00
        </Paragraph>
        <Paragraph>
          {" "}
          The search for gold starts in the Sahara Desert! There’s potential for
          big cash prizes of up to 1,000 your stake with the Free Spins feature
          so you’ll want to stock up on those Story Book Free Spin Symbols to
          get your chance.{" "}
        </Paragraph>
      </div>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
