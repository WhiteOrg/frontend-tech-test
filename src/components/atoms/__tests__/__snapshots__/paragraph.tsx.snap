// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ATOMS: Render testing Should render paragraph 1`] = `
<div>
  <p
    className="sc-AxjAm dHkAvS"
    color="#17dfe7"
    size="small"
  >
    PRIZE POOL
  </p>
  <p
    className="sc-AxjAm DSefD"
    size="medium"
  >
    P.C
  </p>
  <p
    className="sc-AxjAm itERZv"
    color="white"
    size="large"
  >
    £25,000.00
  </p>
  <p
    className="sc-AxjAm qAnnS"
  >
     
    The search for gold starts in the Sahara Desert! There’s potential for big cash prizes of up to 1,000 your stake with the Free Spins feature so you’ll want to stock up on those Story Book Free Spin Symbols to get your chance.
     
  </p>
</div>
`;
